packages/protocol/contracts/SubjectFactory.sol

```diff
@@ -256,6 +266,13 @@ contract SubjectFactory is SecurityModule, ISubjectFactory {
         }
     }
 
+    function _calculateFee(
+        uint256 _amount,
+        uint256 _fee
```

Comment:
Recommend better naming to avoid confusion in the function calls below, e.g.
```suggestion
        uint256 _feeAmount,
        uint256 _feePct
```